BFWControls/Modules/Carousel/Controller/CarouselViewController.swift

```diff
@@ -121,16 +115,17 @@ class CarouselViewController: UICollectionViewController {
                                                 animated: animated)
     }
     
+    private func updatePageControl() {
+        pageControl.frame.origin.x = (collectionView!.bounds.width - pageControl.frame.width) / 2 + collectionView!.contentOffset.x
+        pageControl.frame.origin.y = collectionView!.bounds.height - pageControl.frame.height + collectionView!.contentOffset.y - controlInsetBottom
+        pageControl.currentPage = loopedPageForPage(page + 1)
```

Comment:
There's a bug that's causing all pages to show as `page  - 1`. I've messaged you about it in Slack.